src/features/layers_in_area/atoms/layersGlobalResource.ts

```diff
@@ -2,13 +2,40 @@ import { apiClient } from '~core/apiClientInstance';
 import { createAsyncAtom } from '~utils/atoms/createAsyncAtom';
 import { LAYERS_IN_AREA_API_ERROR } from '../constants';
 import type { LayerInArea } from '../types';
+import {createAtom} from "~utils/atoms";
+import {currentApplicationAtom} from "~core/shared_state";
```

Comment:
For some reason pre-commit hook didn't work for you.
run `npx prettier --write` manually for fix formatting